src/Dashboard/Dashboard_components/PostListHeader_Components/PostForm.jsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import postData from "../../../../js_functions/post";
+import { useEffect } from "react";
+import putData from "../../../../js_functions/put";
+
+const initialForm = {
+  title: "",
+  content: "",
+  contactId: undefined,
+};
+
+export default function PostForm({
+  setReloadPostList,
+  reloadPostList,
+  reloadComments,
+  setReloadComments,
+  post,
+  edit,
+  liRef
+}) {
+
+  const [form, setForm] = useState(initialForm);
+  const titleExtractedFromPost = form.content.slice(0, 20);
+  //allows the form to be set to the value of the comment we want to edit
+  useEffect(() => {
+
+    post && post.title !== titleExtractedFromPost + "..."
+      ? setForm(post)
+      //handles cases where the title had been generated based on post content
+      : post && setForm({ ...post, title: "" });
+        //it wants 'titleExtractedFromPost in the dependencies. The issue with that is that is results in the value of the input being reset to post.content with each click, making modifications impossible. Which is why I ended up getting rid of it. 
```

Comment:
If a ternary needs to be spread across more than one short line, chances are an if statement is easier to read. Always favour readability over condensation, ternary should be used sparingly. I've seen a lot of real world PR's be rejected for over-using them